modules/core/setup.php

```diff
@@ -75,7 +76,8 @@
 add_output('settings', 'list_style_setting', true, 'core', 'mailto_handler_setting', 'after');
 add_output('settings', 'msg_list_icons_setting', true, 'core', 'list_style_setting', 'before');
 add_output('settings', 'delete_prompt_setting', true, 'core', 'list_style_setting', 'after');
-add_output('settings', 'no_password_setting', true, 'core', 'delete_prompt_setting', 'after');
+add_output('settings', 'delete_attachment_setting', true, 'core', 'delete_prompt_setting', 'after');
```

Comment:
Can you put it after: Enable Attachment Reminder probably named enable_attachment_reminder_setting